packages/kbn-optimizer/limits.yml

```diff
@@ -93,7 +93,7 @@ pageLoadAssetSize:
   monitoring: 80000
   navigation: 37269
   newsfeed: 42228
-  observability: 95000
+  observability: 135000
```

Comment:
Is this something we can keep in the async bundle?

Reply:
What cause the page bundle to grow this much is the need to import the `ObservabilityAlertStatusFilter` component and the `getAlertSummaryTimeRange` into the infra plugin, when I tried to import them directly without exposing the exports on common and public folders, the compiler was throwing an error indicating they should be exported as I did.
Not sure if we can do it differently to keep them in the async bundle, or maybe I'm missing a step to easily do it.